### ISSUE TITLE
CONFLICT: python3-netifaces-0.10.9_4 with installed pkg python-netifaces-0.10.9_3

### ISSUE THREAD

**Issue by avex48** [void-linux/void-packages#29162]
Hi,
Conflict or other issues with system upgrades are usually resolved within a day or two.
Unfortunately, I have not been able to update the system for several days due to this conflict:

$ sudo xbps-install -Su
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/nonfree/x86_64-repodata' ...
CONFLICT: python3-netifaces-0.10.9_4 with installed pkg python-netifaces-0.10.9_3 (matched by python-netifaces>=0)
Transaction aborted due to conflicting packages.

Is there a solution?
Thank you.

**Comment by Chocimier** [comment]
python-netifaces-0.10.9_3 is designated to removal in `removed-packages`, there should be no such problem unless you blocked it. Did you?

**Comment by avex48** [comment]
No, but until yesterday trying to remove it would break my Calibre installation.
Now I've removed it, it didn't give me any problems and I was able to upgrade the system!
???
Greetings.
